package.json

```diff
@@ -31,52 +31,55 @@
   },
   "types": "src/openmrs-esm-patient-chart.tsx",
   "devDependencies": {
-    "@babel/core": "^7.5.5",
-    "@babel/plugin-proposal-class-properties": "^7.5.5",
-    "@babel/preset-env": "^7.5.5",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/preset-typescript": "^7.3.3",
+    "@babel/core": "^7.7.2",
+    "@babel/plugin-proposal-class-properties": "^7.7.0",
+    "@babel/preset-env": "^7.7.1",
+    "@babel/preset-react": "^7.7.0",
+    "@babel/preset-typescript": "^7.7.2",
     "@openmrs/esm-error-handling": "^1.1.0",
-    "@openmrs/react-root-decorator": "^1.0.0",
-    "@testing-library/react": "^9.1.3",
-    "@types/jest": "^24.0.18",
-    "@types/react": "^16.9.2",
-    "@types/react-dom": "^16.9.0",
-    "@types/react-router-dom": "^4.3.5",
-    "babel-eslint": "^10.0.2",
+    "@openmrs/react-root-decorator": "^3.0.0",
+    "@testing-library/react": "^9.3.2",
+    "@types/jest": "^24.0.23",
+    "@types/react": "^16.9.11",
+    "@types/react-dom": "^16.9.4",
+    "@types/react-router-dom": "^5.1.2",
+    "@types/webpack-env": "^1.14.1",
+    "babel-eslint": "^10.0.3",
     "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.6",
     "browserslist-config-openmrs": "^1.0.0",
     "clean-webpack-plugin": "^3.0.0",
     "concurrently": "^5.0.0",
     "css-loader": "^3.2.0",
-    "dayjs": "^1.8.16",
-    "eslint": "^6.2.1",
-    "eslint-config-prettier": "^6.1.0",
-    "eslint-config-ts-react-important-stuff": "^1.0.0",
-    "eslint-plugin-prettier": "^3.1.0",
-    "fork-ts-checker-webpack-plugin": "^1.5.0",
-    "husky": "^3.0.4",
+    "dayjs": "^1.8.17",
+    "eslint": "^6.6.0",
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-config-ts-react-important-stuff": "^1.1.0",
+    "eslint-plugin-prettier": "^3.1.1",
+    "fork-ts-checker-webpack-plugin": "^3.1.0",
```

Comment:
See https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/releases - the breaking changes don't affect us.